operations/observability/mixins/cross-teams/dashboards/gitpod-overview.libsonnet

```
@@ -9,6 +9,7 @@ local row = grafana.row;
 local prometheus = grafana.prometheus;
 local template = grafana.template;
 local graphPanel = grafana.graphPanel;
+local barGaugePanel = grafana.barGaugePanel;
```

```suggestion
local barGaugePanel = import '../../components/bar_gauge_panel.libsonnet';
```

This way you don't need the workaround of moving directories inside the vendor folder 🙂

---

Fixed